rdflib/plugins/parsers/ntriples.py

```diff
@@ -135,10 +145,15 @@ def __init__(
         self.buffer: Optional[str] = None
         self.file: Optional[Union[TextIO, codecs.StreamReader]] = None
         self.line: Optional[str] = ""
+        self.ignore_errors = False
```

Comment:
should not be a public variable.

Reply:
Hey, thanks a lot for your reviews. 

Can we make it a global variable instead? Would that suffice?